gooey/tests/test_processor.py

```diff
@@ -9,34 +9,62 @@ class TestProcessor(unittest.TestCase):
     def test_extract_progress(self):

```

Comment:
You'll also need to add an integration(ish) test file which verifies the UI does the right things for the various options available. There's some prior art [here](https://github.com/chriskiehl/Gooey/blob/master/gooey/tests/test_header.py). 

A lot of regressions in Gooey have tended to be on the UI side as it slowly drifts out of sync with the rest of the code.

Reply:
Added integration tests similar to that of the test_header.py. Unsure how to test the onComplete screen though, will look into that